command/tls_ca_create.go

```diff
@@ -61,24 +85,50 @@ CA Create Options:
     Domain of Nomad cluster. Only used in combination with -name-constraint.
     Defaults to "nomad".
 
+  -locality
+    Locality of the CA. Defaults to "San Francisco".
+
   -name-constraint
     Enables the DNS name restriction functionality to the CA. Results in the CA
     rejecting certificates for any other DNS zone. If enabled, localhost and the
     value of -domain will be added to the allowed DNS zones field. If the UI is
     going to be served over HTTPS its hostname must be added with
     -additional-domain. Defaults to false.
+
+  -organization
+    Organization of the CA. Defaults to "HashiCorp Inc.".
+
+  -organizational-unit
+    Organizational Unit of the CA. Defaults to "Nomad".
+
+  -postal-code
+    Postal Code of the CA. Defaults to "94105".
+
+  -province
+    Province of the CA. Defaults to "CA".
+
+  -street-address
+    Street Address of the CA. Defaults to "101 Second Street".
+
 `
 	return strings.TrimSpace(helpText)
 }
 
 func (c *TLSCACreateCommand) AutocompleteFlags() complete.Flags {
 	return mergeAutocompleteFlags(c.Meta.AutocompleteFlags(FlagSetClient),
 		complete.Flags{
-			"-additional-domain": complete.PredictAnything,
-			"-common-name":       complete.PredictAnything,
-			"-days":              complete.PredictAnything,
-			"-domain":            complete.PredictAnything,
-			"-name-constraint":   complete.PredictAnything,
+			"-additional-domain":   complete.PredictAnything,
+			"-common-name":         complete.PredictAnything,
+			"-days":                complete.PredictAnything,
+			"-country":             complete.PredictAnything,
+			"-domain":              complete.PredictAnything,
+			"-locality":            complete.PredictAnything,
+			"-name-constraint":     complete.PredictAnything,
+			"-organization":        complete.PredictAnything,
+			"-organizational-unit": complete.PredictAnything,
+			"-postcode":            complete.PredictAnything,
```

Comment:
This is declared in the help as `post-code` and the flag setup as `postal-code` so will need to be updated along with either of those.